libexec/remake-api-migrations.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# -f so that it's OK when there are no migrations to start with
+rm -rf CreeDictionary/API/migrations/*.py
+touch CreeDictionary/API/migrations/__init__.py
+pipenv run python CreeDictionary/manage.py makemigrations API
+
```

Comment:
this script is called when we rebuilt database, so that API migrations are always up to date with models.py